.cirrus.yml

```diff
@@ -124,6 +124,27 @@ task:
     - name: web_tests-7_last-linux # last Web shard must end with _last
       << : *WEB_SHARD_TEMPLATE
 
+    - name: web_integration_test_linux
+      compile_host_script: |
+        cd $ENGINE_PATH/src
+        ./flutter/tools/gn --unoptimized --full-dart-sdk
+        ninja -C out/host_debug_unopt
+      fetch_framework_script: |
+        cd $ENGINE_PATH/src/flutter/tools
+        ./clone_flutter.sh
+        cd $FRAMEWORK_PATH/flutter
+        bin/flutter update-packages --local-engine=host_debug_unopt
+      script:
```

Comment:
How do you envision adding more integration tests? Are they supposed to be added here as another step in this `script:` section or as a separate entry?

Reply:
I think it depends on how these tests will develop. One thing to note is we are able to run these tests in 6 different browsers over 3 different OS. They might be using the same app to test or different apps.
However the setup is different for them (very different in some settings)
I'm planning to setup a meeting for next weeks to identify critical scenarios. If there are over 20 for Chrome Linux I think I will write script which can also be invoked from LUCI recipes. 
If they all use the same app (let's say something comprehensive such as gallery) we can add a test runner.
On the other hand if they are all scattered between OSes and devices and apps then I am not planning to invest this tooling effort for each.
This example is particularly important to be as readable as possible cause other team members also asked me how to run integration tests from the engine.